dist/mapbox-gl.css

```diff
@@ -303,6 +343,65 @@ a.mapboxgl-ctrl-logo {
     will-change: transform;
 }
 
+.mapboxgl-user-location-dot {
+    background-color: #1DA1F2;
+    width: 20px;
```

Comment:
I think that works, at that size it's pretty much the same size as the icon in the control button which I think is good.